{{cookiecutter.repostory_name}}/bin/rotate-local-backups.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020, Reef Technologies, BSD 3-Clause License
+
+import argparse
+from pathlib import Path
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description='Keep all but N most recent files in a directory')
+    parser.add_argument('file_number', help='How many last files to keep', type=int)
+    return parser.parse_args()
+
+
+def rotate_backups(arguments):
```

Comment:
this function should accept the path and file_number as parameters. You might need to use it in the middle of the night, while the system of the client is on fire - that is not a very good time for refactoring. Lets do it ahead of time :)